In the OpenMPTarget build on the ORNL Jenkins CI
```
In file included from /var/jenkins/workspace/Kokkos_PR-8316/build/core/unit_test/serial/TestSerial_ParallelScanRangePolicy.cpp:2:
/var/jenkins/workspace/Kokkos_PR-8316/core/unit_test/TestParallelScanRangePolicy.hpp:92:10: error: unused variable 'check_scan_results_start2' [-Werror,-Wunused-variable]
   92 |     auto check_scan_results_start2 = [&]() {
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~
/var/jenkins/workspace/Kokkos_PR-8316/core/unit_test/TestParallelScanRangePolicy.hpp:310:7: note: in instantiation of function template specialization '(anonymous namespace)::TestParallelScanRangePolicy<char>::test_scan<>' requested here
  310 |       test_scan<Args...>(work_sizes[i]);
      |       ^
/var/jenkins/workspace/Kokkos_PR-8316/core/unit_test/TestParallelScanRangePolicy.hpp:320:7: note: in instantiation of function template specialization '(anonymous namespace)::TestParallelScanRangePolicy<char>::test_scan<>' requested here
  320 |     f.test_scan<>(work_sizes);
      |       ^

--- a/core/unit_test/TestParallelScanRangePolicy.hpp
+++ b/core/unit_test/TestParallelScanRangePolicy.hpp
@@ -87,6 +87,7 @@ struct TestParallelScanRangePolicy {
       Kokkos::deep_copy(postfix_results, 0);
     };
 
+#ifndef KOKKOS_ENABLE_OPENMPTARGET  // FIXME_OPENMPTARGET
     // Lambda for checking errors from stored value at each index
     // starting from 2.
     auto check_scan_results_start2 = [&]() {
@@ -111,6 +112,7 @@ struct TestParallelScanRangePolicy {
       Kokkos::deep_copy(prefix_results, 0);
       Kokkos::deep_copy(postfix_results, 0);
     };
+#endif
 
     // If policy template args are not given, call parallel_scan()
     // with work_size input, if args are given, call

